packages/web3-eth-ens/test/fixtures/ens/bytecode/NameWrapperBytecode.ts

```diff
@@ -15,6 +15,6 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-// https://github.com/ensdomains/ens-contracts/blob/master/contracts/resolvers/PublicResolver.sol
+// https://github.com/ensdomains/ens-contracts/blob/master/contracts/resolvers/mocks/DummyNameWrapper.sol
```

Comment:
I suggest using the permanent link of the file, so if it was updated later the comment still contains the exact version that this Bytecode belongs to.
```suggestion
// https://github.com/ensdomains/ens-contracts/blob/1b6b4b59003cceb045634d84ab63ae9bc2c0f8c2/contracts/resolvers/mocks/DummyNameWrapper.sol
```

Reply:
Right! Fixed
thanks